### PR TITLE
update bdcom support (mempool, software version, serial number, hw version)

### PR DESCRIPTION
**Pull request by hrtrd**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by hrtrd** [comment]
@Jellyfrog help pls

File: includes/polling/os/bdcom.inc.php [hunk 1]
```diff
@@ -1,13 +1,23 @@
 <?php
```
Comment (Jellyfrog) [inline includes/polling/os/bdcom.inc.php:1]: This should be ported to the new format while you're at it. See https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php#L34 or similar

**Comment by Jellyfrog** [comment]
Probably some new entries for the test data also?

File: includes/polling/os/bdcom.inc.php (hunk 1 above)
Comment (hrtrd) [reply includes/polling/os/bdcom.inc.php:1]: hm, ill try it for new os, today. if I succeed, I will transfer all my OS

File: includes/polling/mempools/bdcom.inc.php [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
+$perc = snmp_get($device, ['.1.3.6.1.4.1.3320.3.6.10.1.12.0'], '-OQv', 'NMS-CHASSIS');
 
-$mempool['free']  = ['bdcomMemoryPoolFree.0'];
-$mempool['used']  = $memory_pool['bdcomMemoryPoolUsed.0'];
-$mempool['total'] = $mempool['free'] + $mempool['used'];
-$mempool['perc']  = $mempool['used'] / $mempool['total'];
+if (is_numeric($perc)) {
+    $mempool['perc'] = $perc;
+    $mempool['used'] = $perc;
+    $mempool['total'] = 100;
```
Comment (murrant) [inline includes/polling/mempools/bdcom.inc.php:18]: memory is meant to be in bytes, not percent...  This will show up as 100 bytes of memory.

Comment (hrtrd) [reply includes/polling/mempools/bdcom.inc.php:18]: bdcom >2019year
#sh mem-util 
 mem util threshold is 90%
 current mem util is 27%
 Status:Normal

Comment (Jellyfrog) [reply includes/polling/mempools/bdcom.inc.php:18]: You sure there is not a some OID containing the value in bytes?

Comment (hrtrd) [reply includes/polling/mempools/bdcom.inc.php:18]: yes. bdcom changed the policy, and now all their systems are brought to the same form. This data is also displayed in the original monitoring software.

Comment (PipoCanaja) [reply includes/polling/mempools/bdcom.inc.php:18]: @hrtrd Not a single place (inventory, entity-mib, etc etc) where an absolute value for the total memory would be provided ?

Comment (hrtrd) [reply includes/polling/mempools/bdcom.inc.php:18]: Screenshot_1356
only this (screenshot). Show usage in percent.  the vendor switched to the new mib libraries. We use now in / opt / librenms / mibs / bdcom already deprecated

**Comment by PipoCanaja** [comment]
In the screen capture, @hrtrd , we can see nmsnvRAMUsed and nmsnvRAMSize. So I would suggest to check those, and you can probably re-create all the missing values from it. 
Bye

**Comment by hrtrd** [comment]
> In the screen capture, @hrtrd , we can see nmsnvRAMUsed and nmsnvRAMSize. So I would suggest to check those, and you can probably re-create all the missing values from it.
> Bye

Nope. This oids only for BDCOM NMS app, for CPE memory/

File: includes/polling/os/bdcom.inc.php (hunk 1 above)
Comment (murrant) [reply includes/polling/os/bdcom.inc.php:1]: @hrtrd actually, better to move it to Yaml now.  See docs: https://docs.librenms.org/Developing/os/Initial-Detection/#yaml-based-os-discovery

**Comment by Jellyfrog** [comment]
@hrtrd maybe remove mempool for now so the rest could be merged?

**Pull request closed without merging**